code/procs/mobprocs/chatprocs.dm

```diff
@@ -7,6 +7,30 @@
 /mob/verb/say_verb(message as text)
 	set name = "say"
 	//&& !src.client.holder
+
+	if (!client.preferences.auto_capitalization)
```

Comment:
Why is this negated? Shouldn't the code trigger when auto_cpitalization is *on*?

Reply:
You’re right, it should. I think I screwed up the definition of auto_capitalization when I first wrote it. Fixing now.